src/components/App.tsx

```diff
@@ -1,19 +1,93 @@
-import React,  { useState }  from 'react';
+import React, { useState, useEffect } from 'react';
 import Header from './Header';
 import MainContent from './MainContent';
 import Footer from './Footer';
 import ErrorBoundary from './ErrorBoundary';
+import { ModalWindowType } from './../model';
+import ModalWindowOpener from './ModalWindowOpener';
+import { Genre } from '../model';
+
+const INITIAL_MOVIE_LIST = [
+  {
+    value: 'DOCUMENTARY 1',
+    genre: Genre.Documentary,
+  },
+  {
+    value: 'COMEDY 1',
+    genre: Genre.Comedy,
+  },
+  {
+    value: 'DOCUMENTARY 2',
+    genre: Genre.Documentary,
+  },
+  {
+    value: 'DOCUMENTARY 3',
+    genre: Genre.Documentary,
+  },
+  {
+    value: 'COMEDY 4',
+    genre: Genre.Comedy,
+  },
+  {
+    value: 'CRIME 2',
+    genre: Genre.Crime,
+  },
+];
+
+export interface Handlers {
+  addMovie: Function;
+  deleteMovie: Function;
+  updateMovie: Function;
+}
 
 const App = () => {
-  const [ searchText, setSearchText ] = useState('');
+  const [searchText, setSearchText] = useState('');
+  const [modalWindowType, setModalWindowType] = useState(ModalWindowType.None);
+  const [movies, setMovies] = useState([]);
+  const [selectedMovie, setSelectedMovie] = useState('');
+
+  useEffect(() => {
+    setMovies(INITIAL_MOVIE_LIST);
+  }, []);
```

Comment:
This happens on the initial component mount. In this case you can remove useEffect at all and just pass INITIAL_MOVIE_LIST to the useState instead of blank array at 46 line